libs/tools/shareable-examples/src/utils/get-example-module.ts

```diff
@@ -113,9 +108,10 @@ export function createTransformer(
     }
 
     /** Visit an importDeclaration and remove imports if necessary. */
-    function visitImportDeclaration(node: ImportDeclaration): Node | undefined {
-      const importFrom = isStringLiteral(node.moduleSpecifier)
-        ? node.moduleSpecifier.text
+    function visitImportDeclaration(node: Node): VisitResult<Node> {
+      const importDeclaration = node as ImportDeclaration;
```

Comment:
Seems a tiny bit odd to me. Would this always work? (e.g If `node` isn't of type `ImportDeclaration`)

Reply:
I guess the check is done before the function call to determine that you should call the visitImport declaration but you are right I would add the ImportDeclaration type to the function arguments